extra/command-not-found.bash

```diff
@@ -5,6 +5,20 @@ command_not_found_handle () {
   set +o verbose
 
   mapfile -t pkgs < <(pkgfile -bv -- "$cmd" 2>/dev/null)
+
+  
+  if [[ ${#pkgs[@]} -eq 1 ]] && [[ -n "$PKGFILE_PROMPT_INSTALL_MISSING" ]]; then
+    pkg=$(echo ${pkgs[0]} | awk '{print $1}')
+    read -r -p "Install $pkg? [Y/n] " response
+    response=${response,,}
+    if [[ $response =~ ^(no|n)$ ]]; then
```

Comment:
This is bad -- entering any garbage which isn't "no" or "n" will mean "yes, install the package".

I would have expected something closer to:

```
[[ -z $response || $response = [Yy] ]] || return 0
```